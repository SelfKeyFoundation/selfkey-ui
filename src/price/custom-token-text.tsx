import * as React from 'react';
// @ts-ignore
import injectSheet, { StyleSheet, StyledComponentProps } from 'react-jss';

const styles: StyleSheet = {
    customTokenText: {
        fontFamily: 'ProximaNovaRegular',
        fontSize: '14px',
        color: '#93b0c1',
        paddingTop: '8px',
        paddingBottom: '50px',
        textAlign: 'center'
    }
};

export const CustomTokenText = injectSheet(styles)<{}>(({classes, children}) => (
    <div className={classes.customTokenText}>
        {children}
    </div>
));

export default CustomTokenText;
