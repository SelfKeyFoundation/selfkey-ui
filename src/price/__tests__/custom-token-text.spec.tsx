import * as React from 'react';
import { create } from 'react-test-renderer';
import { CustomTokenText } from '../custom-token-text';


describe('Custom Token Text', () => {
  it('should show Custom Token Text', () => {
    const component = create(
      <CustomTokenText>Send or receive any custom ERC-20 token.</CustomTokenText>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});





