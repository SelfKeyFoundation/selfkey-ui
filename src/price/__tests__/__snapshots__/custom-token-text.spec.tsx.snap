// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom Token Text should show Custom Token Text 1`] = `
<div
  className="Component-customTokenText-0-2-1"
>
  Send or receive any custom ERC-20 token.
</div>
`;
